x-pack/plugins/transform/common/api_schemas/type_guards.ts

```diff
@@ -56,6 +57,15 @@ export const isDeleteTransformsResponseSchema = (
   );
 };
 
+export const isResetTransformsResponseSchema = (
+  arg: unknown
+): arg is ResetTransformsResponseSchema => {
+  return (
+    isPopulatedObject(arg) &&
+    Object.values(arg).every((d) => isPopulatedObject(d, ['transformReset']))
```

Comment:
Will there ever be a case a response has a mix of transformReset and something else? Can we use `.some` here? Just curious not a suggestion.

Reply:
The current APIs we have don't allow these combinations at the moment. Depending on the endpoint, this attribute is hard coded for example like `transformDeleted` or `transformReset`.